src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -85,6 +85,14 @@ public void disengageRatchet() {
         rightServo.setPosition(0);
     }
 
+    public SparkLimitSwitch getLeftRetractedLimitSwitch() {
```

Comment:
Let's not expose the limit switches outside the climber and add logical functions to expose the states we are interested in (see the command comment)